frontend/src/services/protectedemail.service.ts

```diff
@@ -0,0 +1,30 @@
+export interface CsvStatus {
```

Comment:
call it protected.service.ts, should be easily extendable to sms or other channels